connector/dialect.tdd

```diff
@@ -524,192 +296,53 @@
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='datetime' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='date'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='date' />
-    </function>
-    <function group='aggregate' name='MIN' return-type='bool'>
-      <formula>BOOL_AND(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='bool' />
-    </function>
-    <function group='aggregate' name='MIN' return-type='real'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='logical' name='ISNULL' return-type='bool'>
+      <formula>(%1 IS NULL)</formula>
       <argument type='real' />
     </function>
-    <function group='aggregate' name='MIN' return-type='int'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='logical' name='ISNULL' return-type='bool'>
+      <formula>(%1 IS NULL)</formula>
       <argument type='int' />
     </function>
-    <function group='aggregate' name='MIN' return-type='str'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='logical' name='ISNULL' return-type='bool'>
+      <formula>(%1 IS NULL)</formula>
       <argument type='str' />
     </function>
-    <function group='aggregate' name='MIN' return-type='datetime'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='logical' name='ISNULL' return-type='bool'>
+      <formula>(%1 IS NULL)</formula>
       <argument type='datetime' />
     </function>
-    <function group='aggregate' name='MIN' return-type='date'>
-      <formula>MIN(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='date' />
-    </function>
-    <function group='aggregate' name='STDEV' return-type='real'>
-      <formula>STDDEV(%1)</formula>
-      <unagg-formula>NULL</unagg-formula>
+    <function group='numeric;logical' name='ZN' return-type='real'>
+      <formula>COALESCE(%1, 0)</formula>
       <argument type='real' />
     </function>
-    <function group='aggregate' name='STDEVP' return-type='real'>
-      <formula>(CASE WHEN COUNT(%1) = 1 THEN 0.0 WHEN COUNT(%1) &gt; 0 THEN (STDDEV(%1) * SQRT(COUNT(%1) - 1) / SQRT(COUNT(%1)) ) ELSE NULL END)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN NULL ELSE 0.0 END)</unagg-formula>
-      <argument type='real' />
+    <function group='numeric;logical' name='ZN' return-type='int'>
+      <formula>COALESCE(%1, 0)</formula>
+      <argument type='int' />
     </function>
-    <function group='aggregate' name='SUM' return-type='real'>
-      <formula>SUM(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='numeric' name='DIV' return-type='int'>
+      <formula>CASE WHEN %2 = 0 THEN NULL ELSE ( %1 / %2 ) END</formula>
```

Comment:
Why are we using this formula when the semantics of MongoSQL's `/` operator already do this for us?

Reply:
Updated the formula to use `/` only